docs/en/core/defaults-files.md

```diff
@@ -113,13 +156,18 @@ self-contained: true
  etc. 
 ```
 
-More options for the writer can be found [here](https://pandoc.org/MANUAL.html#options-affecting-specific-writers-1), including options for numbered headings, the table of contents, etc. 
+Zettlr will search for the reference document in the same directory as the Markdown files that you're exporting. If you want different formatting for different projects and files, it can be convenient to have a custom-reference.docx within each of those folders. 
```

Comment:
Really? Never tried this myself.

Reply:
Yes it definitely works! I have a different custom-reference.docx for each project, and as long as they're in the correct folder, Zettlr uses the right one, which is really convenient for all the different types of reports I need.